storage/bucket.go

```diff
@@ -93,6 +93,43 @@ func (yig *YigStorage) SetBucketAcl(bucketName string, policy datatype.AccessCon
 	return nil
 }
 
+func (yig *YigStorage) SetBucketLogging(bucketName string, bl datatype.BucketLoggingStatus,
+	credential common.Credential) error {
+	helper.Logger.Info("enter SetBucketLogging")
+	bucket, err := yig.MetaStorage.GetBucket(bucketName, true)
+	if err != nil {
+		return err
+	}
+	if bucket.OwnerId != credential.UserId {
+		return ErrBucketAccessForbidden
+	}
```

Comment:
I believe bucket info could be fetched in API layer by means of https://github.com/journeymidnight/yig/blob/9a74b969e98983047e7303dcf28b0104e289cf34/api/object-handlers.go#L91-L95
So check bucket owner there. Same for `GetBucketLogging`.